ts/src/base/Exchange.ts

```diff
@@ -4606,6 +4606,25 @@ export default class Exchange {
         return [ value, params ];
     }
 
+    /**
+     * @param {object} params - extra parameters
+     * @param {object} request - existing dictionary of request
+     * @param {string} exchangeSpecificKey - the key for chain id to be set in request
+     * @param {object} currencyCode - (optional) existing dictionary of request
+     * @param {boolean} isRequired - (optional) whether that param is required to be present
+     * @returns {object[]} - returns [request, params] where request is the modified request object and params is the modified params object
+     */
+    handleRequestNetwork (params: any, request: any, exchangeSpecificKey: string, currencyCode:Str = undefined, isRequired: boolean = false) {
```

Comment:
can we type params and request with `Dict` ?

Reply:
ah sure, done,

Reply:
can this be reviewed today? tnx